serverless/messaging/reference-content/migration.mdx

```diff
@@ -76,11 +76,11 @@ Scaleway has automatically migrated each of your NATS namespaces and their crede
 
 The migration should have been seamless: if you logged into the Scaleway console before migration/GA you saw the "old" version of Messaging and Queuing with your existing resources showing as namespaces. After migration/GA, you now see the "new" version with your newly migrated resources now showing as accounts.
 
-In terms of the API, the [old `v1alpha1`](https://www.scaleway.com/en/developers/api/messaging-and-queuing/) version continues to be available for a 30 day period, starting from the date of General Availability, and co-existing with the new [`v1beta1`](https://www.scaleway.com/en/developers/api/messaging-and-queuing/nats-api/) version. The documentation for both is accessible via the [Scaleway Developers website](https://www.scaleway.com/en/developers/api). 
+In terms of the API, the [old `v1alpha1`](https://www.scaleway.com/en/developers/api/messaging-and-queuing/) version continues to be available until the 1st of December 2023. After this date it will be retired and you must used the new [`v1beta1`](https://www.scaleway.com/en/developers/api/messaging-and-queuing/nats-api/) version. The documentation for both is accessible via the [Scaleway Developers website](https://www.scaleway.com/en/developers/api). 
```

Comment:
```suggestion
In terms of the API, the [old `v1alpha1`](https://www.scaleway.com/en/developers/api/messaging-and-queuing/) version continues to be available until the 1st of December 2023. After this date it will be retired and you must use the new [`v1beta1`](https://www.scaleway.com/en/developers/api/messaging-and-queuing/nats-api/) version. The documentation for both is accessible via the [Scaleway Developers website](https://www.scaleway.com/en/developers/api). 
```